packages/utilities/src/dom.ts

```diff
@@ -86,8 +86,8 @@ export function getParent(child: HTMLElement, allowVirtualParents: boolean = tru
  * Gets the elements which are child elements of the given element.
  * If `allowVirtualChildren` is `true`, this method enumerates virtual child elements
  * after the original children.
- * @param parent
- * @param allowVirtualChildren
+ * @param parent - The element to get the children of.
```

Comment:
Is the - between the parameter and the rest of the line something introduced by API Extractor? The jsdoc standard doesn't require it and in fact Typescript doesn't require it. Including it will result in the `-` superfluously showing up in intellisense

Reply:
Yes, and TSDoc will require the hyphen in "strict" mode.  Omitting the hyphen would be ambiguous and error-prone, e.g. considering the other optional type annotations that JSDoc permits (which are not needed in TypeScript, but commonly occur nonetheless in TypeScript code bases).

I believe the IntelliSense you're referring to is coming from the VS Code plugin, not the TypeScript compiler itself.  If you strongly object to the hyphen, you could open a TSDoc issue to discuss it. Otherwise I'd suggest to open a GitHub issue for VS Code; their doc comment analysis already has a number of other known issues -- this was part of the motivation for the compiler group getting involved with TSDoc.